src/main/kotlin/no/nav/arbeidsgiver/altinnrettigheter/proxy/klient/AltinnrettigheterProxyKlient.kt

```diff
@@ -95,28 +95,23 @@ class AltinnrettigheterProxyKlient(
         }
         when (result) {
             is Result.Failure -> {
-                if (!response.isClientError && !response.isServerError) {
-                    throw AltinnrettigheterProxyException(
-                            ProxyErrorUtenResponse(
-                                    "Feil ved bruk av Altinnrettigheter proxy pga " +
-                                            "'${result.getException().message}'",
-                                    ProxyError.Kilde.ALTINN_RETTIGHETER_PROXY)
-                    )
-                }
-
                 val proxyErrorMedResponseBody = ProxyErrorMedResponseBody.parse(
                         response.body().toStream(),
```

Comment:
Burde kanskje ta høyde for at body ikke finnes?

Reply:
Var det det du mente ?